examples/BlazorServerApp/Data/WeatherForecastService.cs

```diff
@@ -0,0 +1,20 @@
+namespace BlazorServerApp.Data
+{
+    public class WeatherForecastService
+    {
+        private static readonly string[] Summaries = new[]
+        {
+            "Freezing", "Bracing", "Chilly", "Cool", "Mild", "Warm", "Balmy", "Hot", "Sweltering", "Scorching"
```

Comment:
Seems a bit confusing that the summary is random, because when I run the example I can get a temperature of 100 but it's Freezing.

Reply:
The WeatherForecast part is from the Blazor template.